apps/trading/pages/index.page.tsx

```diff
@@ -20,15 +20,18 @@ const MARKETS_QUERY = gql`
   }
 `;
 
-const marketList = ({ markets }: MarketsLanding) =>
-  orderBy(
+const marketList = ({ markets }: MarketsLanding) => {
+  const filteredMarkets =
```

Comment:
suggestion (non blocking): I quite like using lodash's .chain or .flow methods to make things a little more readable.